util/util.go

```diff
@@ -37,3 +55,58 @@ func RunWithRetry(ctx context.Context, retryCnt int, interval time.Duration, f f
 	}
 	return errors.Trace(err)
 }
+
+// IsFileExist returns true if the file exists.
+func IsFileExist(name string) bool {
+	_, err := os.Stat(name)
+	return err == nil
+}
+
+// Wget downloads a string URL to the dest directory and returns the file path.
+// SKips if the file already exists.
+func Wget(ctx context.Context, rawURL string, dest string) (string, error) {
+	u, err := url.Parse(rawURL)
+	if err != nil {
+		return "", err
+	}
+
+	if len(dest) == 0 {
+		dest = "."
+	}
+
+	fileName := path.Base(u.Path)
+	filePath := path.Join(dest, fileName)
+	if IsFileExist(filePath) {
+		return filePath, nil
+	}
+
+	os.MkdirAll(dest, 0755)
+	err = exec.CommandContext(ctx, "wget", "--tries", "20", "--waitretry", "60",
+		"--retry-connrefused", "--dns-timeout", "60", "--connect-timeout", "60",
+		"--read-timeout", "60", "--directory-prefix", dest, rawURL).Run()
+	return filePath, errors.Annotatef(err, "wget %s to %s", rawURL, dest)
+}
+
+// InstallArchive downloads the URL and extracts the archive to the dest diretory.
+// Supports zip, and tarball.
+func InstallArchive(ctx context.Context, rawURL string, dest string) error {
```

Comment:
What's the purpose of this function? Do we need to download some package?

Reply:
Yeah, Sqllogic test need to download a cases archive.